boards/adafruit/qt_py_esp32s3/doc/index.rst

```diff
@@ -0,0 +1,287 @@
+.. zephyr:board:: adafruit_qt_py_esp32s3
+
+Overview
+********
+
+An Adafruit based Xiao compatible board based on the ESP32-S3, which is great
+for IoT projects and prototyping with new sensors.
+
+For more details see the `Adafruit QT Py ESP32S3`_ product page.
+
+Hardware
+********
+
+This board comes in 2 variants, both based on the ESP32-S3 with WiFi and BLE
+support. The default variant supporting 8MB of flash with no PSRAM, while the
+``psram`` variant supporting 4MB of flash with 2MB of PSRAM. Both boards have a
+USB-C port for programming and debugging and is based on a standard XIAO 14
+pin pinout.
+
+In addition to the Xiao compatible pinout, it also has a RGB NeoPixel for
+status and debugging, a reset button, and a button for entering the ROM
+bootloader or user input. Like many other Adafruit boards, it has a
+`SparkFun Qwiic`_-compatible `STEMMA QT`_ connector for the I2C bus so you
+don't even need to solder.
+
+ESP32-S3 is a low-power MCU-based system on a chip (SoC) with integrated
+2.4 GHz Wi-Fi and Bluetooth® Low Energy (Bluetooth LE). It consists of
+high-performance dual-core microprocessor (Xtensa® 32-bit LX7), a low power
+coprocessor, a Wi-Fi baseband, a Bluetooth LE baseband, RF module, and
+numerous peripherals.
+
+Supported Features
+==================
+
+Current Zephyr's Adafruit QT Py ESP32-S3 board supports the following features:
+
++------------+------------+-------------------------------------+
+| Interface  | Controller | Driver/Component                    |
++============+============+=====================================+
+| UART       | on-chip    | serial port                         |
++------------+------------+-------------------------------------+
+| GPIO       | on-chip    | gpio                                |
++------------+------------+-------------------------------------+
```

Comment:
i2s missed here also

Reply:
This board is almost completely copied from seeed_xiao_esp32s3, and its missing from there too. I'll add it to this board, but we should add it to the Xiao board if that truely is the case.